### DISCUSSION TITLE
Spectre.Console markup for lolcat

### DISCUSSION THREAD

**Discussion by si618** [spectreconsole/spectre.console#1047] [Show and tell]
Hi,

Just released a new [dotnet lolcat library](https://github.com/si618/lolcat) with support for Spectre.Console style markup. It's a silly but fun project that some might like to play with.

No idea if there would be enough interest in bringing it into Spectre.Console proper; colour animation might be nice to add to status text?

**Comment by patriksvensson** [comment]
@si618 Very cool stuff!

Appreciate you asking, but I think it's a bit too niche to belong in the Spectre.Console library.

**Reply by si618** [reply]
Cheers!

And no worries at all, it's quite esoteric and has a hint of `<blink>` about it :)

**Comment by gakera** [comment]
Cool! I was looking for gradient support in a bunch of stuff, but I didn't get this far haha 
https://github.com/spectreconsole/spectre.console/discussions/1034

**Comment by wbaldoumas** [comment]
@si618 if not directly added to Spectre.Console, I'd vouch for introducing further Spectre.Console support to lolcat directly. I particularly am interested in color animation for things like FigletText.
